src/hoa-renderer.js

```diff
@@ -47,12 +47,12 @@ function HOARenderer(context, options) {
   this._renderingMode = 'ambisonic';
   this._ambisonicOrder = 3;
 
-  if (options) {
-    if (options.HRIRUrl)
+  if (options !== undefined) {
```

Comment:
if an argument (or all arguments) are omitted, they will be undefined, this allows us to use the defaults in their place.